### DISCUSSION TITLE
Unauthorized while trying to translate a field

### DISCUSSION THREAD

**Discussion by tombreit** [wagtail/wagtail-localize#406] [Q&A] [answered]
In an non-production wagtail installation I get an ``Unauthorized`` exception even when logged in as a superuser:

```
[08/Jun/2021 17:04:01] "GET /admin/pages/13/edit/ HTTP/1.1" 200 21405
[08/Jun/2021 17:04:01] "GET /admin/localize/jsi18n/ HTTP/1.1" 200 3187
Unauthorized: /admin/localize/translate/5/strings/11/edit/
[08/Jun/2021 17:04:10] "PUT /admin/localize/translate/5/strings/11/edit/ HTTP/1.1" 401 58
```

when trying to translate a field: 

![grafik](https://user-images.githubusercontent.com/1015232/121213393-76d46e80-c87e-11eb-88fc-cafd574c4227.png)

I'm sure I've translated some pages successfully some time ago...

I'm now on wagtail 2.13.1 and wagtail-localize 0.9.5, but this error was already present in 2.12.x

**Comment by tombreit** [comment, marked as answer]
Sorry for the noise - this issue was caused by my root rest_framework settings:


```
# project settings.py

REST_FRAMEWORK = {
    ...
    'DEFAULT_AUTHENTICATION_CLASSES': [
         'rest_framework.authentication.TokenAuthentication',
    ]
}
```

After moving/deleting this setting fields could be translated again.

**Reply by kaedroho** [reply]
No worries, useful to know about this, thanks for posting your resolution!
